Description: Allow package to build with glibc-2.16
 'gets' has been removed from glibc-2.16 but gnulib still refers to it
 causing build failures.
 The simple fix for now is just to put a check around the reference in
 lib/stdio.in.h
 A new release made with a new gnulib is a better fix.
 .
Author: Wookey <wookey@wookware.org>
Last-Update: 2012-11-15
Bug : http://lists.gnu.org/archive/html/bug-gnulib/2012-03/msg00186.html
Bug-Debian: http://bugs.debian.org/687986
Forwarded: no

Index: cpio-2.11+dfsg/gnu/stdio.in.h
===================================================================
--- cpio-2.11+dfsg.orig/gnu/stdio.in.h
+++ cpio-2.11+dfsg/gnu/stdio.in.h
@@ -136,10 +136,11 @@ _GL_WARN_ON_USE (fflush, "fflush is not
 #endif
 
 /* It is very rare that the developer ever has full control of stdin,
-   so any use of gets warrants an unconditional warning.  Assume it is
-   always declared, since it is required by C89.  */
+   so any use of gets warrants an unconditional warning. */
 #undef gets
+#ifdef HAVE_RAW_DECL_GETS
 _GL_WARN_ON_USE (gets, "gets is a security hole - use fgets instead");
+#endif
 
 #if @GNULIB_FOPEN@
 # if @REPLACE_FOPEN@
